.github/husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
Correct me if I'm wrong, but I think this file needs to be under the root `.husky` directory, like [here](https://github.com/hirosystems/gh-actions-example/tree/main/.husky) for Husky to pick it up correctly.

Reply:
with `husky install .github/husky` we can install husky whereever. i typically do this to not clutter the root dir, as it tends to explode for js projects already and treat `/.github` as a `.config` directory...

but on second thought i would just remove husky from this repo, seems overkill here

Reply:
Husky is only being used to help configure commit-hooks for the repo to run `commitlint` on each commit to ensure the message conforms to [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). Otherwise invalid commit messages may cause issues with the release process.
So if you are able to set up commit-hooks in the `.git` dir without Husky, that should work too.